_posts/2020-06-12-coding-with-kids.md

```diff
@@ -0,0 +1,195 @@
+
+We also tried once to go further and check if we could find something from [CodinGame](https://www.codingame.com/) that we could do. Unfortunately it seems that the challenges there were still a bit too hard. Maybe we get back there in few years. 
+
+
+## Creating the first home page - the hard way
+
+Last and maybe the least is making the home pages. I was once told that the easiest way to make home pages is to make a word document and then save it as a HTML file. If you know anything about home pages, you might know that there are plenty of problems that will arise from that approach. Instead, I taught my kids the same painful route that I took 25 years ago. Basically we opened up a notepad and then into another screen the [w3schools](https://www.w3schools.com/html/default.asp). Then we made an index document using the simplest possible syntax with having title with two words and the body with a sentence. We hosted it in Azure Blob Storage with static websites feature enabled that allows you to serve static html files from the blog storage. I had to check out what it has costed me, and I figured out that less than 1c. Seems like a cheap way to start. 
+
+Needless to say this was kind of boring approach and maybe we come back to this later. Now the website is "under construction" like all the webpages back in the nineties. 
+
+## Afterwords
+
+In summary my kids knew to how to program with few different languages before school. You do not need to know how to read to start programming, because it can be done with drawing (Sphero). ScratchJr also is able to teach "try and learn" type of approach, which does not require necessary any reading skills. Later on, Python is a great choice for first "real programming language" as it is widely supported and easy to learn. 
+
+There are some supporting sources that I did not mention earlier. [Code.org](https://code.org/) has been used at school and seems to have great deal of material for learning coding. There is also section for younglings starting from age of four. Hello Ruby books by Linda Liukas have also been loved by our children, and I feel that they have also helped my children on learning to think like a programmer. Then there is a board game called RoboGem, which has been played both in school and at home, which definitely teaches you how to give instructions for machines. Hard recommendations for all three learning source mentioned. 
```

Comment:
to have great deal -> to have a great deal
helped my children on -> helped my children in
Hard recommendations -> Strong recommendations